AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingViewController.swift

```diff
@@ -88,6 +91,31 @@ class MeetingViewController: UIViewController {
             layout.scrollDirection = .vertical
         }
     }
+    
+    private func registerForAppLifeCycleNotification() {
+        NotificationCenter.default.addObserver(self, selector: #selector(appMovedToBackground), name: UIApplication.didEnterBackgroundNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(appMovedToForeground), name: UIApplication.willEnterForegroundNotification, object: nil)
+    }
+    
+    private func deregisterForAppLifeCycleNotification() {
+        NotificationCenter.default.removeObserver(self, name: UIApplication.didEnterBackgroundNotification, object: nil)
+        NotificationCenter.default.removeObserver(self, name: UIApplication.willEnterForegroundNotification, object: nil)
+    }
+    
+    @objc private func appMovedToBackground() {
+        isLocalOn = meetingModel?.isLocalVideoActive ?? false
```

Comment:
Can we just do `isLocalOn = meetingModel?.isLocalVideoActive`?

Reply:
updated